package.json

```diff
@@ -14,7 +14,7 @@
     "lint:js:fix": "eslint src --fix",
     "patch:hosts": "fec patch-etc-hosts",
     "start": "HOT=true fec dev",
-    "test": "jest",
+    "test": "jest --no-cache",
```

Comment:
I'm ok doing this temporarily, but we want the cache in general: https://jestjs.io/docs/cli#--cache